_chapters/chapter_03.md

```diff
@@ -562,6 +595,14 @@ end
 {% endlist %}
 {% endsteps %}
 
+{% aside %}
+### Arrays 
+
+You might remember reading about arrays in the [Ruby in 100 minutes](http://tutorials.jumpstartlab.com/projects/ruby_in_100_minutes.html#7.-arrays) tutorial. Arrays are most commonly used for storing a list or collection of things.
+
+You can also read more about them and the methods you can use with them in the [Ruby documentation](http://ruby-doc.org/core-2.3.1/Array.html).  
```

Comment:
It's not an array - its an ActiveRecord Collection .... find a way to keep it simple but not be misleading.